db/controller/commentsId.controller.js

```diff
@@ -0,0 +1,21 @@
+const { insertCommentByArticleId } = require("../models/commentsId.model");
+exports.postCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
```

Comment:
consider that we should now be using our error handlers to send out the error responses - to make sure they are set up properly to handle this, refer back to the error handling notes